src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,80 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+
+type Props = {
+  onMovies: (m: Movie | null) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({
+  onMovies,
+}) => {
+  const [title, setTitle] = useState('');
+  const [selectedMovie, setSelectedMovie] = useState<Movie | null>(null);
+  const [hasFindError, setHasFindError] = useState(false);
+  const [hasLoadingSubmit, setHasLoadingSubmit] = useState(false);
+  const [hasFirstLoad, setHasFirstLoad] = useState(true);
+
+  const createNewMovie = (movieDate: MovieData) => {
+    const newDescrip = movieDate.Plot === 'N/A'
+      ? 'Unfortunately there is no description'
+      : movieDate.Plot;
+    const newposter = movieDate.Poster === 'N/A'
+      ? 'https://via.placeholder.com/360x270.png?text=no%20preview'
+      : movieDate.Poster;
+
+    return {
+      title: movieDate.Title,
+      description: newDescrip,
+      imgUrl: newposter,
+      imdbId: movieDate.imdbID,
+      imdbUrl: `https://www.imdb.com/title/${movieDate.imdbID}`,
+    };
+  };
+
+  const handlerTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setHasFindError(false);
+  };
+
+  const handlerSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    setHasLoadingSubmit(true);
+
+    getMovie(title)
+      .then(res => {
+        if ('Response' in res && res.Response === 'False') {
```

Comment:
Checking for 'Response' in the response object might not be the best way to handle errors. Consider checking the status code of the response or use a try-catch block to handle promise rejections.